### ISSUE TITLE
SDL init error with Flatpak Tauon

### ISSUE THREAD

**Issue by Bauke** [Taiko2k/TauonMusicBox#282]
I run Ubuntu and recently did a `do-release-upgrade` to 19.10, and just now did a proper `apt upgrade` which had a ton of packages to update, as well as a `flatpak update` that included some more updates, so I don't exactly know where to even begin with this issue but:

When I start Tauon with Flatpak, it starts but doesn't render anything. Running with `--verbose` gives me:

```
SDL init error: Couldn't find matching GLX visual
SDL render error: Invalid renderer
```

If I run Tauon from source, it works fine. So I must be missing some package causing SDL to fail, but I have no clue what it could be. Searching online hasn't gotten me anywhere, so here I am.

If this issue ends up being impossible to solve I'm fine with switching over to running from source instead of Flatpak, I just figured I'd create an issue before I start copying configuration over. :P

**Comment by Taiko2k** [comment]
You don't update to 20.04?

Anyway yeah that sounds like a headache. An issue with Flatpak/runtimes and video drivers most likely. (You use nvidia?).

I'd say it'll probably resolve itself down the line, or you could create/check for an issue... not sure where though.

**Comment by Bauke** [comment]
I had an issue with another program on 18.04 that was easily fixed by upgrading to 19.10, I'll probably upgrade to 20.04 soon though.

Yeah, I use Nvidia (`nvidia-driver-440` according to Ubuntu), I tried `flatpak repair` but all seems good there. So now I don't know what to do. :P Guess I'll just use the source version from now on, maybe if I encounter another program that'll fail like this I can find out something more.

I'll close this issue for now and if I happen to figure out a solution I'll come back and add it here.

**Comment by NSQY** [comment]
Running into this with 470.74
Waiting for https://github.com/flathub/org.freedesktop.Platform.GL.nvidia/pull/72 to be merged...
